src/workers/parser/tests/mapExpression.spec.js

```diff
@@ -294,6 +301,20 @@ describe("mapExpression", () => {
         originalExpression: false
       }
     },
+    {
+      name: "await (with SyntaxError)",
+      expression: "await new Promise())",
+      newExpression: formatAwait("await new Promise())"),
+      parseExpression: false,
```

Comment:
what's this doing?

Reply:
we don't want to parse the expression like for the other expression, because it would throw (see https://github.com/devtools-html/debugger.html/pull/7695/files/07ef9a76c3f1c5dbed0dfbdd096786dcc15344bd#diff-fff0d7c8c37a491198a0e4f041c09ea5L23) 
So we need to explicitly indicate that we don't want to parse it. And then, we check that an invalid expression (note the extra closing parens) is still wrapped in an async iife.